games-action/openclonk-7.0

```diff
@@ -0,0 +1,128 @@
+SUMMARY="Free multiplayer action game"
```

Comment:
According Gentoo portage it should be : "A free multiplayer action game where you control clonks"
https://gpo.zugaina.org/games-action/openclonk